v2/oauth/multiple-frontends-with-separate-backends.mdx

```diff
@@ -0,0 +1,427 @@
+
+# Multiple frontend domains with separate backends
+
+You can use the following guide if you have a single [**Authentication Service**](/docs/oauth/introduction#authentication-service) that is used by multiple applications.
+In turn, each app has separate **`frontend `** and **`backend`** instances that are served from different domains.
+
+Using the actual **OAuth 2.0** terminology, we can say that each **`frontend`** can be considered a [**Client**](/docs/oauth/introduction#client) and each **`backend`** is a [**Resource Server**](/docs/oauth/introduction#resource-server).
+
+:::info
+This guide assumes that you already have setup and configured **SuperTokens** in your [**Authentication Service**](/docs/oauth/introduction#authentication-service). 
+
+For more information on how to do that please check our [quickstart guide](/docs/thirdparty/introduction).
+:::
+
+## 1. Create the OAuth2 Clients
+
+For each of your **`frontend`** applications you will have to create a separate [**OAuth2 client**](/docs/oauth/introduction#client).
+This can be done by directly calling the **SuperTokens Core** API.
+
+<CoreInjector defaultValue="https://example.com">
+
+```bash
+curl -X POST ^{coreInjector_uri_without_quotes}/recipe/oauth2/admin/clients \
+  -H "Content-Type: application/json" \
+  -H "api-key: ^{coreInjector_api_key_without_quotes}" \
+  -d '{
+      "client_name": "<YOUR_CLIENT_NAME>",
+      "response_types": ["code", "id_token"],
+      "grant_types": ["authorization_code", "refresh_token"],
+      "redirect_uri": ["https://<YOUR_DOMAIN>.com/oauth/callback"]
+    }'
+```
+
+</CoreInjector>
+
+
+- `client_name` - A human-readable name of the client that will be presented to the [**end-user**](/docs/oauth/introduction#authorization) during authorization.
+- `response_types` - Specifies the types of responses your client expects from the Authorization Server.
+    - `code`: Indicates that the [**Client**](/#authorization) will receive an [**Authorization Code**](/#authorization) that will be exchanged for an [**Access Token**](/#authorization).
+    - `id_token`: Indicates that the [**Client**](/#authorization) expects an [**ID Token**](/#authorization)
+- `grant_types` - The grant types that the [**Client**](/#authorization) will use. 
+    - `authorization_code`: Allows exchanging the [**Authorization Code**](/#authorization) for an [**Access Token**](/#authorization).
+    - `refresh_token`: Allows exchanging the [**Refresh Token**](/#authorization) for a new [**Access Token**](/#authorization).
+- `redirect_uri` - A list of URIs to which the [**Authorization Server**](/#authorization) will send the user-agent (browser) after completing the authorization step
+
+If the creation was successful, the response will return a `201` status code.
+The response body will contain the **`client_id`** and the `client_secret` fields. Safe them for each request because we will need to use them in the next steps.
```

Comment:
```suggestion
The response body will contain the **`client_id`** and the `client_secret` fields. Save them for each request because we will need to use them in the next steps.
```

Reply:
resolved